src/main/java/org/vietabroader/view/MainView.java

```diff
@@ -89,6 +91,8 @@ public void setText(String text) {
     private final MessageLabel lblSheetMessage = new MessageLabel(" ");
     private final JComboBox<String> cbbSheet = new JComboBox<>();
     private final JProgressBar prgIndicator = new JProgressBar();
+    private final HashMap<String, String> columnArray = new HashMap<>();
+    private final SheetFreshController sheetFreshController = new SheetFreshController();
```

Comment:
Instead of doing this, you should move all the related components to this scope like I did, then define the controller and set its components in [`initControllers()`](https://github.com/vietabroader/VAQR/blob/master/src/main/java/org/vietabroader/view/MainView.java#L387)

It's hard to read and error-prone if you scatter the method calls like that.